"Add a Pallet" tutorial nicks pallet `devhub/latest` tag breaks the build

Using the dependency as documented in the tutorial:

```
[dependencies.pallet-nicks]
default-features = false
git = 'https://github.com/paritytech/substrate.git'
tag = 'devhub/latest'
version = '4.0.0-dev'
```

Command `cargo check -p node-template-runtime` fails with a lot of errors, an excerpt:

```
error[E0277]: the trait bound `sp_runtime::generic::UncheckedExtrinsic<MultiAddress<AccountId32, ()>, Call, MultiSignature, (CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckEra<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeTransactionPayment<Runtime>)>: Decode` is not satisfied
     --> /Users/kukabi/Documents/development/substrate/substrate-node-template/runtime/src/lib.rs:399:31
      |
  399 |         fn offchain_worker(header: &<Block as BlockT>::Header) {
      |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Decode` is not implemented for `sp_runtime::generic::UncheckedExtrinsic<MultiAddress<AccountId32, ()>, Call, MultiSignature, (CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckEra<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeTransactionPayment<Runtime>)>`
      |
      = help: the following implementations were found:
                <sp_runtime::generic::UncheckedExtrinsic<Address, Call, Signature, Extra> as Decode>
      = note: required because of the requirements on the impl of `Codec` for `sp_runtime::generic::UncheckedExtrinsic<MultiAddress<AccountId32, ()>, Call, MultiSignature, (CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckEra<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeTransactionPayment<Runtime>)>`
      = note: required because of the requirements on the impl of `sp_api::BlockT` for `sp_runtime::generic::Block<sp_runtime::generic::Header<u32, sp_runtime::traits::BlakeTwo256>, sp_runtime::generic::UncheckedExtrinsic<MultiAddress<AccountId32, ()>, Call, MultiSignature, (CheckSpecVersion<Runtime>, CheckTxVersion<Runtime>, CheckGenesis<Runtime>, CheckEra<Runtime>, CheckNonce<Runtime>, CheckWeight<Runtime>, ChargeTransactionPayment<Runtime>)>>`
```

`devhub/latest` tag is also in conflict with the 3rd paragraph after the dependency declaration, which explains the monthly release naming convention:

> The remaining lines specify that Cargo should use the `pallet-nicks` crate from the `paritytech/substrate` repository with the commit tag identified using the `monthly-YYYY-MM` naming convention.

--- a/v3/how-to-guides/02-pallet-design/g-loose-coupling/index.mdx
+++ b/v3/how-to-guides/02-pallet-design/g-loose-coupling/index.mdx
@@ -36,7 +36,7 @@ pallet you want to couple to accordingly:
 
 ```toml
 [dependencies]
-frame-support = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-08', version = '4.0.0-dev' }
+frame-support = { default-features = false, git = 'https://github.com/paritytech/substrate.git', tag = 'devhub/latest', version = '4.0.0-dev' }
 
 # -- snip
 
@@ -115,7 +115,7 @@ construct_runtime! (
   the [Identity pallet](https://github.com/paritytech/substrate/blob/master/frame/identity/src/lib.rs#L149-L151) and its use of its
   [specific weighting methods](https://github.com/paritytech/substrate/blob/master/frame/identity/src/weights.rs#L46-L64)
 - the [`KeyOwnerProofSystem` trait](/rustdocs/latest/frame_support/traits/trait.KeyOwnerProofSystem.html)
-  [used in `pallet_granpa`](https://github.com/paritytech/substrate/blob/master/frame/grandpa/src/lib.rs#L106)
+  [used in `pallet_grandpa`](https://github.com/paritytech/substrate/blob/master/frame/grandpa/src/lib.rs#L106)
 
 ## Resources
 

--- a/v3/how-to-guides/03-weights/b-benchmarking/index.mdx
+++ b/v3/how-to-guides/03-weights/b-benchmarking/index.mdx
@@ -24,7 +24,7 @@ category: weights
     {
       title: 'Overview',
       description: `
-This guide steps throught the process of adding benchmarking to a pallet and runtime.
+This guide steps through the process of adding benchmarking to a pallet and runtime.
 In addition, it covers the steps of writing a simple benchmark for a pallet as well as testing and running the benchmarking tool.
 This guide does not cover updating weights with benchmarked values.
       `,
@@ -136,7 +136,7 @@ benchmarking.
    [dependencies.pallet-you-created]
    default-features = false
    path = '../pallets/pallet-you-created'
-   version = '3.0.0-monthly-2021-08'
+   version = '4.0.0-dev'
 
    [features]
    runtime-benchmarks = [

--- a/v3/how-to-guides/04-ocw/a-transactions/index.mdx
+++ b/v3/how-to-guides/04-ocw/a-transactions/index.mdx
@@ -40,7 +40,7 @@ For example:
     ///
     /// By implementing `fn offchain_worker` you declare a new offchain worker.
     /// This function will be called when the node is fully synced and a new best block is
-    /// succesfuly imported.
+    /// successfully imported.
     /// Note that it's not guaranteed for offchain workers to run on EVERY block, there might
     /// be cases where some blocks are skipped, or for some the worker runs twice (re-orgs),
     /// so the code should be able to handle that.
@@ -122,7 +122,7 @@ You will need to initialize this account in the next step.
     // `send_signed_transaction()` return type is `Option<(Account<T>, Result<(), ()>)>`. It is:
     //   - `None`: no account is available for sending transaction
     //   - `Some((account, Ok(())))`: transaction is successfully sent
-    //   - `Some((account, Err(())))`: error occured when sending the transaction
+    //   - `Some((account, Err(())))`: error occurred when sending the transaction
     let results = signer.send_signed_transaction(|_account| {
       Call::on_chain_call { key: val }
     });
@@ -201,7 +201,7 @@ You will need to initialize this account in the next step.
     - Sign the raw payload with the account public key.
     - Finally, bundle all data up and return a tuple of the call, the caller, its signature, and any signed extension data.
 
-  You can see a full example of the code in the [**Substrate** code base](https://github.com/paritytech/substrate/blob/monthly-2021-12/bin/node/runtime/src/lib.rs#L960-L999).
+  You can see a full example of the code in the [**Substrate** code base](https://github.com/paritytech/substrate/blob/devhub/latest/bin/node/runtime/src/lib.rs).
 
 8. Implement `SigningTypes` and `SendTransactionTypes` in the runtime to support submitting transactions, whether they are signed or unsigned.
 
@@ -220,10 +220,10 @@ You will need to initialize this account in the next step.
   }
   ```
 
-  You can see an example of this implementation in the [**Substrate** code base](https://github.com/paritytech/substrate/blob/monthly-2021-12/bin/node/runtime/src/lib.rs#L1001-L1012).
+  You can see an example of this implementation in the [**Substrate** code base](https://github.com/paritytech/substrate/blob/devhub/latest/bin/node/runtime/src/lib.rs#L1001-L1012).
 
 9. Inject an account for this pallet to own. 
-  In a developemnt environment (node running with `--dev` flag), this account key is inserted in the `node/src/service.rs` file as follows:
+  In a development environment (node running with `--dev` flag), this account key is inserted in the `node/src/service.rs` file as follows:
 
   ```rust
   pub fn new_partial(config: &Configuration) -> Result <SomeStruct, SomeError> {
@@ -300,7 +300,7 @@ To enable Substrate to accept certain unsigned transactions, you must implement 
 
   In this example, users can call the on-chain `extrinsic1` function without a signature, but not any other extrinsics.
 
-  To see a full example of how `ValidateUnsigned` is implemented in a pallet, refer to [`pallet-ocw` in **Substrate Off-chain Worker Demo**](https://github.com/jimmychu0807/substrate-offchain-worker-demo/blob/v2.0.0/pallets/ocw/src/lib.rs#L209-L240) and [`pallet-example-offchain-worker` in **Substrate**](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker/src/lib.rs#L297-L325).
+  To see a full example of how `ValidateUnsigned` is implemented in a pallet, refer to [`pallet-ocw` in **Substrate Off-chain Worker Demo**](https://github.com/jimmychu0807/substrate-offchain-worker-demo/blob/v2.0.0/pallets/ocw/src/lib.rs#L209-L240) and [`pallet-example-offchain-worker` in **Substrate**](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker/src/lib.rs#L297-L325).
 
 2. In the off-chain worker function, you can send unsigned transactions as follows:
 
@@ -344,7 +344,7 @@ To enable Substrate to accept certain unsigned transactions, you must implement 
 
 4. Implement the `SendTransactionTypes` trait for the runtime as described in [sending signed transactions](#sending-signed-transactions).
 
-You can see a full example in [`pallet-example-offchain-worker` in **Substrate** code base](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker).
+You can see a full example in [`pallet-example-offchain-worker` in **Substrate** code base](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker).
 
 ## Sending unsigned transactions with signed payloads
 
@@ -376,7 +376,7 @@ The differences between sending unsigned transactions and sending unsigned trans
   }
   ```
 
-  You can also see an example [here](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker/src/lib.rs#L344-L357).
+  You can also see an example [here](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker/src/lib.rs#L344-L357).
 
 1. In your pallet's `offchain_worker` function, call the signer, then the function to send the transaction:
 
@@ -394,7 +394,7 @@ The differences between sending unsigned transactions and sending unsigned trans
       //   The returned result means:
       //   - `None`: no account is available for sending transaction
       //   - `Some((account, Ok(())))`: transaction is successfully sent
-      //   - `Some((account, Err(())))`: error occured when sending the transaction
+      //   - `Some((account, Err(())))`: error occurred when sending the transaction
       if let Some((_, res)) = signer.send_unsigned_transaction(
         // this line is to prepare and return payload
         |acct| Payload { number, public: acct.public.clone() },
@@ -454,7 +454,7 @@ The differences between sending unsigned transactions and sending unsigned trans
 
   This example uses [`SignedPayload`](https://paritytech.github.io/substrate/latest/frame_system/offchain/trait.SignedPayload.html) to verify that the public key in the payload has the same signature as the one provided.
 
-Refer to the [off-chain function call](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker/src/lib.rs#L505-L532) and [the implementation of `ValidateUnsigned`](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker/src/lib.rs#L308-L318) for a working example of the above.
+Refer to the [off-chain function call](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker/src/lib.rs#L505-L532) and [the implementation of `ValidateUnsigned`](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker/src/lib.rs#L308-L318) for a working example of the above.
 
 
 You have now seen how you can use off-chain workers to send data for on-chain storage using:
@@ -465,7 +465,7 @@ You have now seen how you can use off-chain workers to send data for on-chain st
 
 ## Examples
 
-- [Substrate Offchain Worker Example Pallet](https://github.com/paritytech/substrate/tree/monthly-2021-12/frame/examples/offchain-worker)
+- [Substrate Offchain Worker Example Pallet](https://github.com/paritytech/substrate/tree/devhub/latest/frame/examples/offchain-worker)
 - [OCW Pallet in Substrate Offchain Worker Demo](https://github.com/jimmychu0807/substrate-offchain-worker-demo/tree/v2.0.0/pallets/ocw)
 
 ## Related material

--- a/v3/how-to-guides/04-ocw/b-http-requests/index.mdx
+++ b/v3/how-to-guides/04-ocw/b-http-requests/index.mdx
@@ -44,7 +44,7 @@ The Substrate HTTP library supports the following methods:
   ```rust
   // We want to keep the offchain worker execution time reasonable, so we set a hard-coded
   // deadline to 2s to complete the external call.
-  // You can also wait idefinitely for the response, however you may still get a timeout
+  // You can also wait indefinitely for the response, however you may still get a timeout
   // coming from the host machine.
   let deadline = sp_io::offchain::timestamp().add(Duration::from_millis(2_000));
   ```
@@ -102,6 +102,6 @@ The Substrate HTTP library supports the following methods:
 
 ## Examples
 
-- [**Offchain Worker Example Pallet** in Substrate](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker/src/lib.rs#L568-#L621)
+- [**Offchain Worker Example Pallet** in Substrate](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker/src/lib.rs#L568-#L621)
 - [**OCW Pallet** in Substrate Offchain Worker Demo](https://github.com/jimmychu0807/substrate-offchain-worker-demo/blob/master/pallets/ocw/src/lib.rs#L363-#L401)
 - [**Offchain HTTP source** in Substrate Core](https://github.com/paritytech/substrate/blob/master/primitives/runtime/src/offchain/http.rs#L63-L76)

--- a/v3/how-to-guides/04-ocw/c-local-storage/index.mdx
+++ b/v3/how-to-guides/04-ocw/c-local-storage/index.mdx
@@ -25,7 +25,7 @@ Nonetheless there are also times when the data is not suitable to be saved on-ch
 This includes temporary data or intermediate calculations that can be discarded once the computation is completed.
 
 In this guide, we will instruct off-chain workers to write data to the local storage without passing the data among the whole blockchain network.
-The concept of a **Storage Lock** is introduced to have value accessible consistently among mutliple OCWs.
+The concept of a **Storage Lock** is introduced to have value accessible consistently among multiple OCWs.
 OCWs are asynchronously run at the end of each block production and they are not limited by how long they run, so at any point of time there could be multiple OCW instances being initiated as shown below.
 
 ![Multiple OCW Instances](../../../../src/images/how-to-guides/ocw/local-storage/multiple-ocws.png)
@@ -35,7 +35,7 @@ The local storage APIs are similar to their on-chain counterpart, using `get`, `
 This is done as a single atomic operation.
 The atomicity guarantees that the new value is calculated based on up-to-date information; if the value had been updated by another thread in the meantime, the write would fail.
 
-Note that as values stored in local storage has not gone through the consensus mechanism among the network, so they are subject to manupulation of the node operator.
+Note that as values stored in local storage has not gone through the consensus mechanism among the network, so they are subject to manipulation of the node operator.
 
 In this how-to guide, we will first check from the storage, which acts as a cache, if the computed value exists.
 If the cached value is found, off-chain worker returns; else it will try to acquire a lock, perform the intensive computation, and save it to the storage/cache.
@@ -165,5 +165,5 @@ If the cached value is found, off-chain worker returns; else it will try to acqu
 
 ## Examples
 
-- [**Off-chain worker example pallet** in Substrate](https://github.com/paritytech/substrate/blob/monthly-2021-12/frame/examples/offchain-worker/src/lib.rs#L374-L437)
+- [**Off-chain worker example pallet** in Substrate](https://github.com/paritytech/substrate/blob/devhub/latest/frame/examples/offchain-worker/src/lib.rs#L374-L437)
 - [**OCW pallet** demo](https://github.com/jimmychu0807/substrate-offchain-worker-demo/blob/master/pallets/ocw/src/lib.rs#L299-L342)

--- a/v3/how-to-guides/09-tools/a-integrate-try-runtime/index.mdx
+++ b/v3/how-to-guides/09-tools/a-integrate-try-runtime/index.mdx
@@ -34,7 +34,7 @@ which steps through which dependencies to include and where to include them in o
 <Message
   type={`red`}
   title={`Warning`}
-  text={`Be sure to use the latest \`monthly-*\` tag when adding your dependencies.`}
+  text={`Be sure to use the \`devhub/latest\` tag when adding your dependencies.`}
 />
 
 ## Steps
@@ -47,8 +47,8 @@ Add the FRAME dependency:
 
 ```rust
 [dependencies]
-frame-try-runtime = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-07', optional = true }
-try-runtime-cli = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-07', optional = true }
+frame-try-runtime = { git = 'https://github.com/paritytech/substrate.git', tag = 'devhub/latest', optional = true }
+try-runtime-cli = { git = 'https://github.com/paritytech/substrate.git', tag = 'devhub/latest', optional = true }
 
 /* --snip-- */
     std = [
@@ -96,8 +96,8 @@ try-runtime = [
 ]
 
 /* --snip-- */
-frame-try-runtime = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-07', optional = true }
-try-runtime-cli = { git = 'https://github.com/paritytech/substrate.git', tag = 'monthly-2021-07', optional = true }
+frame-try-runtime = { git = 'https://github.com/paritytech/substrate.git', tag = 'devhub/latest', optional = true }
+try-runtime-cli = { git = 'https://github.com/paritytech/substrate.git', tag = 'devhub/latest', optional = true }
 /* --snip-- */
 
 ```

--- a/v3/tutorials/02-proof-of-existence/index.mdx
+++ b/v3/tutorials/02-proof-of-existence/index.mdx
@@ -233,7 +233,7 @@ To add the `sp-std` crate to the pallet:
    [dependencies.sp-std]
    default-features = false
    git = 'https://github.com/paritytech/substrate.git'
-   tag = 'monthly-2021-11-1'  # or the latest monthly
+   tag = 'devhub/latest'      # or another tag that is **uniform** for you project
    version = '4.0.0-dev'      # or the latest version
    ```
 

--- a/v3/tutorials/07-add-a-pallet/index.mdx
+++ b/v3/tutorials/07-add-a-pallet/index.mdx
@@ -89,7 +89,7 @@ To add the Nicks pallet to the Substrate node template:
 
    It is important to note that the Substrate runtime compiles to both a native Rust [`std`](https://doc.rust-lang.org/std/) binary and a [WebAssembly (Wasm)](https://webassembly.org/) binary. For more information about compiling [`std`](https://doc.rust-lang.org/std/) and [`no_std`](https://rust-embedded.github.io/book/intro/no-std.html) features, see the [Rust Embedded Book](https://docs.rust-embedded.org/book/intro/no-std.html#summary).
 
-   The remaining lines specify that Cargo should use the `pallet-nicks` crate from the `paritytech/substrate` repository with the commit tag identified using the `monthly-YYYY-MM` naming convention.
+   The remaining lines specify that Cargo should use the `pallet-nicks` crate from the `paritytech/substrate` repository with the commit tag identified using the `devhub/latest` tag.
 
    For an introduction to editing `Cargo.toml` files, see the [Cargo reference documentation](https://doc.rust-lang.org/cargo/reference/index.html).
 

